site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -51,57 +54,63 @@ a list containing dictionaries of type [Ruleset][3], as shown below.
 }
 ```
 
-### Rule Resources {: #manifest-rule-resources }
+## Concepts and Usage {: #concepts-and-usage }
 
-Rules are specified in JSON files referenced under the `"rule_resources"` manifest key.
+To use declarative net requests specify one or more rulesets. A ruleset contains an array of rules. A single rule does one of the following:
```

Comment:
A rule can actually do five things (e.g upgrade requests). Should we say something that makes it clear this is not an exhaustive list?